metadata.json

```diff
@@ -26,7 +26,7 @@
   "requirements": [
     {
       "name": "pe",
-      "version_requirement": ">=3.7.0 <4.0.0"
+      "version_requirement": ">=3.0.0 <=4.3.1"
```

Comment:
So is the Ruby handling the only aspect keeping this module from being Puppet 4 compatible?

Furthermore, does your patch introduce compat w/ 3.x before 3.7?